test/deploy/rackhd_package_install.py

```diff
@@ -29,9 +27,9 @@
 
 # set proxy if required
 PROXYVARS = ''
-if fit_common.fitproxy() and fit_common.fitproxy['url'] != '':
-    PROXYVARS = "export http_proxy=" + fit_common.fitproxy()['url'] + ";" + \
-              "export https_proxy=" + fit_common.fitproxy()['url'] + ";"
+if fit_common.fitproxy()['host'] != '':
+    PROXYVARS = "export http_proxy=http://" + fit_common.fitproxy()['host'] + ":" + fit_common.fitproxy()['port'] + ";" + \
+                "export https_proxy=http://" + fit_common.fitproxy()['host'] + ":" + fit_common.fitproxy()['port'] + ";"
```

Comment:
Can you add a comment describing this?  I think this might confuse other developers working in the code.  They might try to change the http to https.